monetio/sat/_omps_l3_mm.py

```diff
@@ -0,0 +1,72 @@
+def read_OMPS_l3(files):
```

Comment:
Could use a general name like `open_dataset` here (if applicable).

Reply:
Each satellite will likely require different opening methodology so I think we need to specify in the naming what type of file reading is occurring.

Reply:
Module scoping does allow the names to be the same, e.g. the MM model reader functions are called simply `open_mfdataset`.